server/app/views/v1/grid_services/_grid_service.json.jbuilder

```diff
@@ -33,9 +33,9 @@ json.stack do
   json.id grid_service.stack.to_path
   json.name grid_service.stack.name
 end
-json.links grid_service.grid_service_links.map{|s|
-  { id: s.linked_grid_service.to_path, alias: s.alias, name: s.linked_grid_service.name }
-}
+json.links grid_service.grid_service_links.map { |s|
+  { id: s.linked_grid_service.to_path, alias: s.alias, name: s.linked_grid_service.name } if s.linked_grid_service
```

Comment:
Yes, it just hits here harder because we are just removing services from db.. if client is polling rest api at the same time this starts to throw exceptions (no transactions).